### DISCUSSION TITLE
Backing up sql database

### DISCUSSION THREAD

**Discussion by sinnbeck** [postalserver/postal#2434] [Help with using Postal]
Has anyone figured out a clean way of dumping the database to an sql file for backup?

I assumed one of the docker services had `mysqldump` but so far no luck finding it. I hope I dont need to install sql directly on the server just for this :)

**Comment by willpower232** [comment]
Postal leaves database backups up to you (or whoever is managing your database server).

There are also potentially many databases depending on how many Servers you have created within Postal so might not be straightforward.

**Reply by sinnbeck** [reply]
Yeah that's what what I figured. I just hoped that mysqldump or mariabackup was installed in one of the containers. I manage the server myself :)

I will keep looking around and update once I figure it out. At least port 3306 is exposed to the os
